repo-agent/k8s/rbac.generated.yaml

```diff
@@ -11,6 +11,8 @@ rules:
   verbs:
   - get
   - list
+  - patch
```

Comment:
The permissions `patch` and `update` on `secrets` are quite broad. Is it possible to restrict these permissions to specific secret name ?  something like `resourceNames` field in the RBAC role.

Reply:
These permission are necessary for updating the secretes w/ the refresh tokens.  The PR is just added patch and update for this reason